### PR TITLE
feat(alpha): waitlist

### PR DESCRIPTION
**Pull request by bokuhe**
Added waitlist feature.
This is only used in the alpha version and will be removed in the future.
I made it to fetch the config from the assets repository.
https://github.com/palmapp-xyz/assets/tree/main/mobile/alpha

@lwj010 check below
## asset files 
config.json -> whether to use waitlist
waitlist.json -> waitlist address

### CODE REVIEW

File: src/palm-react-native-ui-kit/screens/auth/MainAccountScreen.tsx [hunk 1]
```diff
@@ -24,71 +26,171 @@ const MainAccountScreen = (): ReactElement => {
...
+  //         >
+  //           <FormImage source={images.import} size={32} />
+  //           <FormText style={{ textAlign: 'center' }}>
+  //             {t('Auth.ImportWallet')}
+  //           </FormText>
+  //         </TouchableOpacity>
+  //       </Row>
+  //     </FormBottomSheet>
+  //   </>
+  // )
```
Comment (denniswon) [inline src/palm-react-native-ui-kit/screens/auth/MainAccountScreen.tsx:193]: what is this?

File: src/palm-react-native-ui-kit/screens/auth/RecoverAccountScreen.tsx [hunk 2]
```diff
@@ -99,6 +125,71 @@ const RecoverAccountScreen = (): ReactElement => {
...
+  //         }
+
+  //         return Promise.resolve()
+  //       },
+  //       cancel: () => {
+  //         navigation.pop()
+  //       },
+  //     })
+  //   }
+  // }
```
Comment (denniswon) [inline src/palm-react-native-ui-kit/screens/auth/RecoverAccountScreen.tsx:191]: ?

File: src/palm-react/hooks/app/useAlphaTest.ts [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    } catch (error) {
+      clearTimeout(timeoutId)
+      throw error
+    }
+  }
+
+  const fetchConfig = async (): Promise<void> => {
+    try {
+      const ret = await fetchWithTimeout(
+        'https://raw.githubusercontent.com/palmapp-xyz/assets/main/mobile/alpha/config.json',
```
Comment (denniswon) [inline src/palm-react/hooks/app/useAlphaTest.ts:40]: what is config vs waitlist?

File: src/palm-react/hooks/app/useAlphaTest.ts [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+type AlphaConfig = {
+  waitlist: boolean
+}
+
+/* Hook used only in alpha test
+ * 1. fetch config.json from github assets
+ * 2. fetch waitlist.json from github assets
+ * 3. return config and waitlist
+ */
+const useAlphaTest = (): {
```
Comment (denniswon) [inline src/palm-react/hooks/app/useAlphaTest.ts:13]: useAlphaTest? doesn't get the naming. Maybe just useWaitlist?

**Review by denniswon — CHANGES_REQUESTED** [review]
clean up commented out codes please

File: src/palm-react/hooks/app/useAlphaTest.ts (hunk 3 above)
Comment (bokuhe) [reply src/palm-react/hooks/app/useAlphaTest.ts:40]: config.json -> whether to use waitlist
waitlist.json -> waitlist address

File: src/palm-react/hooks/app/useAlphaTest.ts (hunk 4 above)
Comment (bokuhe) [reply src/palm-react/hooks/app/useAlphaTest.ts]: renamed

File: src/palm-react-native-ui-kit/screens/auth/MainAccountScreen.tsx (hunk 1 above)
Comment (bokuhe) [reply src/palm-react-native-ui-kit/screens/auth/MainAccountScreen.tsx]: I did this to revert the code after the alphatest was over.
But it won't matter delete it.

File: src/palm-react-native-ui-kit/screens/auth/RecoverAccountScreen.tsx (hunk 2 above)
Comment (bokuhe) [reply src/palm-react-native-ui-kit/screens/auth/RecoverAccountScreen.tsx]: I did this to revert the code after the alphatest was over.
But it won't matter delete it.

**Review by denniswon — APPROVED** [review]

**Pull request merged**